release/release.js

```diff
@@ -63,6 +63,7 @@ const confirmDocumentationCorrect = async () => {
   console.log(' - CONFLUENCE')
   console.log('   - Review and update "feature matrix", if necessary.')
   console.log('   - Review and update the relevant subpages under the SDK (Web & Mobile) section of the Onfido product documentation')
+  console.log('     (URL: https://wiki.onfido.co.uk/pages/viewpage.action?pageId=14071325)')
```

Comment:
Should we have links to our wiki in a public repository?

Reply:
Oh, yes. Fair point. I'll remove it.